src/core/benches/gather.rs

```diff
@@ -0,0 +1,62 @@
+use std::fs::File;
+use std::io::BufReader;
+use std::path::PathBuf;
+
+use sourmash::collection::Collection;
+use sourmash::signature::Signature;
+use sourmash::sketch::Sketch;
+use sourmash::{index::calculate_gather_stats, storage::SigStore};
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+
+fn gather_stats_benchmarks(c: &mut Criterion) {
+    let mut filename = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    filename.push("../../tests/test-data/track_abund/47.fa.sig");
+    let file = File::open(filename).unwrap();
+    let reader = BufReader::new(file);
+    let mut sigs: Vec<Signature> = serde_json::from_reader(reader).expect("Loading error");
+    let orig_query = if let Sketch::MinHash(mh) = &sigs.swap_remove(0).sketches()[0] {
+        mh.clone()
+    } else {
+        unimplemented!()
+    };
```

Comment:
Let's put the new `.minhash()` method to use =]

```suggestion
    let sig = Signature::from_reader(reader).expect("Loading error").swap_remove(0);                                                                                                  
    let orig_query = sig.minhash().unwrap();
```